app/assets/src/components/views/discovery/mapping/base_map.scss

```diff
@@ -10,8 +10,12 @@
     position: absolute;
     bottom: 50px;
     right: 15px;
+    box-shadow: 0 0 0 1px rgba(0, 0, 0, 0.1);
 
     button {
```

Comment:
Should this use a class instead? Or is part of 3rd party lib?

Reply:
yes and yes (assuming you meant the button) 👍